### DISCUSSION TITLE
Welcome to eFirebase Discussions!

### DISCUSSION THREAD

**Discussion by rafael-figueiredo-alves** [rafael-figueiredo-alves/eFirebase#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by euclas** [comment]
Olá Rafael.
Precisa colocar uma indicação de quais versões do Delphi/Lázarus funciona.
Essa seria minha 1a. pergunta.

Euclides Franco de Rezende.
euclas@gmail.com
